### PR TITLE
QOL: Dev 1.9 - Regroup Advanced Quality Options

### PR DESCRIPTION
**Pull request by igiannakas**
Felt that the "advanced section" under "quality" needed a bit of a regrouping and cleanup, especially after the addition of all the new features in 1.8 and 1.9 which made it grow quite a bit and difficult to read in a glance :) 

Also renamed the bridge flow and internal bridge flow to include the word "ratio" for consistency purposes with the remaining flow settings. Finally some more tweaks for consistency purposes.

This PR is a proposal on how this may look

**Before:**

image

**After:** (updated to include all commits in this PR)

image

### CODE REVIEW

**Comment by Eldenroot** [comment]
Nice!

**Comment by igiannakas** [comment]
Updated PR with some extra tweaks:

image

**Comment by Eldenroot** [comment]
Maybe this one should start with a capital letter? Srry CZE language.

image

**Comment by igiannakas** [comment]
Done - updated and also improved tool tips and the wall order label.

image

File: src/libslic3r/PrintConfig.cpp [hunk 1]
```diff
@@ -2914,14 +2914,14 @@ def = this->add("filament_loading_speed", coFloats);
     def->set_default_value(new ConfigOptionString("{input_filename_base}_{filament_type[initial_tool]}_{print_time}.gcode"));
 
     def = this->add("make_overhang_printable", coBool);
-    def->label = L("Make overhang printable");
+    def->label = L("Make overhangs printable");
```
Comment (SoftFever) [inline src/libslic3r/PrintConfig.cpp:2917]: Same here, let's update message ids for such small tweaks too.

File: src/libslic3r/PrintConfig.cpp [hunk 2]
```diff
@@ -806,7 +806,7 @@ void PrintConfigDef::init_fff_params()
     def->set_default_value(new ConfigOptionFloat(1));
 
     def = this->add("internal_bridge_flow", coFloat);
-    def->label = L("Internal bridge flow");
+    def->label = L("Internal bridge flow ratio");
```
Comment (SoftFever) [inline src/libslic3r/PrintConfig.cpp:809]: Same here, let's update message ids for such small tweaks too.

**Comment by igiannakas** [comment]
@SoftFever  I think its done :)

**Comment by igiannakas** [comment]
Looks like the build is crashing when opening. Investigating.

**Comment by igiannakas** [comment]
Fixed - incorrectly transferred one option from another PR here hence the GUI was crashing. 

PR is ready to merge.

File: localization/i18n/OrcaSlicer.pot [hunk 3]
```diff
@@ -7726,7 +7726,7 @@ msgid "Internal solid infill pattern"
 msgstr ""
 
 msgid ""
-"Line pattern of internal solid infill. if the detect nattow internal solid "
+"Line pattern of internal solid infill. if the detect narrow internal solid "
```
Comment (SoftFever) [inline localization/i18n/OrcaSlicer.pot:7729]: @igiannakas I update this message as well.

**Review by SoftFever — APPROVED** [review]

**Pull request merged**